Bottom: 2b939086e15468a6b632bb66d060610315caf277
Top:    f9bd6782a32758368b3891aeb78b7fda16253b49
Author: scott snyder <sss@karma>
Date:   2019-07-16 23:31:46 -0400

Refresh of nrf52

---

diff --git a/Adafruit_SPIDevice.h b/Adafruit_SPIDevice.h
index e9597f8..3525795 100644
--- a/Adafruit_SPIDevice.h
+++ b/Adafruit_SPIDevice.h
@@ -20,7 +20,7 @@ typedef enum _BitOrder {
 #endif
 
 // Some platforms have a BitOrder enum but its named MSBFIRST/LSBFIRST
-#if defined(ARDUINO_ARCH_SAMD) || defined(__SAM3X8E__)
+#if defined(ARDUINO_ARCH_SAMD) || defined(__SAM3X8E__) || defined(ARDUINO_ARCH_NRF52)
   #define SPI_BITORDER_MSBFIRST MSBFIRST
   #define SPI_BITORDER_LSBFIRST LSBFIRST
 #endif
